src/Uno.UI/UI/Xaml/Controls/BitmapIcon.cs

```diff
@@ -67,14 +67,6 @@ private void RefreshImage()
 		{
 #if !NET461
 			UpdateImageMonochromeColor();
-
-			if (UriSource != null)
-			{
-				// Force a reload
-				var tmp = UriSource;
-				UriSource = null;
-				UriSource = tmp;
-			}
```

Comment:
This part is needed for reloading on other targets. We may need to adjust this specifically for webassembly.